manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Data Migration Tool

```

```suggestion
PackageName: Samsung Data Migration
```
This must match control Panel

---

Please update file name and path to match this change.